Framework/DataHandling/src/LoadFullprofResolution.cpp

```diff
@@ -654,13 +620,31 @@ void LoadFullprofResolution::parseBankLine(string line, double &cwl,
       string candidate = v[i];
       boost::algorithm::trim(candidate);
       if (!candidate.empty()) {
-        cwl = atof(candidate.c_str());
+        cwl = parseDoubleValue(candidate);
         break;
       }
     }
   }
 }
 
+double LoadFullprofResolution::parseDoubleValue(const std::string &value,
```

Comment:
Does this need to be a member function? Would it be better as a static function inside the implementation file?